modes/longitudinal/src/toolbarButtons.js

```diff
@@ -39,36 +42,36 @@ export default [
       commandName: 'setToolActive',
       commandOptions: { toolName: 'Wwwc' },
       commands: {
-        'windowLevelPreset1': {
-          commandName: 'windowLevelPreset1',
-          commandOptions: {},
+        1: {
+          commandName: 'setWindowLevel',
+          commandOptions: windowLevelPresets[1],
         },
-        'windowLevelPreset2': {
-          commandName: 'windowLevelPreset2',
-          commandOptions: {},
+        2: {
+          commandName: 'setWindowLevel',
+          commandOptions: windowLevelPresets[2],
         },
-        'windowLevelPreset3': {
-          commandName: 'windowLevelPreset3',
-          commandOptions: {},
+        3: {
+          commandName: 'setWindowLevel',
+          commandOptions: windowLevelPresets[3],
         },
-        'windowLevelPreset4': {
-          commandName: 'windowLevelPreset4',
-          commandOptions: {},
+        4: {
+          commandName: 'setWindowLevel',
+          commandOptions: windowLevelPresets[4],
         },
-        'windowLevelPreset5': {
-          commandName: 'windowLevelPreset5',
-          commandOptions: {},
+        5: {
```

Comment:
Does this work? I though objects couldn't have `number` keys?

Reply:
It will always convert to a string, the code works